apps/avifdec.c

```diff
@@ -176,21 +180,38 @@ int main(int argc, char * argv[])
                 return 1;
             }
             imageDimensionLimit = (uint32_t)value;
+        } else if (arg[0] == '-') {
+            fprintf(stderr, "ERROR: unrecognized flag %s\n\n", arg);
```

Comment:
Nit: Say "option" instead of "flag" to match the `syntax()` function?

Also in avifenc.c.